core/src/main/java/com/linecorp/armeria/common/metric/MoreNamingConventions.java

```diff
@@ -18,53 +18,29 @@
 
 import static java.util.Objects.requireNonNull;
 
-import java.util.regex.Pattern;
-
-import javax.annotation.Nullable;
-
-import com.google.common.base.CaseFormat;
-import com.google.common.base.Splitter;
-
-import io.micrometer.core.instrument.Meter.Type;
 import io.micrometer.core.instrument.MeterRegistry;
 import io.micrometer.core.instrument.Metrics;
 import io.micrometer.core.instrument.composite.CompositeMeterRegistry;
 import io.micrometer.core.instrument.config.NamingConvention;
-import io.micrometer.core.instrument.dropwizard.DropwizardMeterRegistry;
-import io.micrometer.prometheus.PrometheusMeterRegistry;
-import io.micrometer.prometheus.PrometheusNamingConvention;
 
 /**
  * Provides commonly-used {@link NamingConvention}s.
  */
 public final class MoreNamingConventions {
+    private static final NamingConvention IDENTITY_NAMING_CONVENTION = ( name, type, baseUnit ) -> name;
```

Comment:
How about:

- Reverting all the changes in this class
- Deprecating this class
- Remove any references to the methods in this class at all in other places

..? It's because we do not even need to touch `MeterRegistry.config().namingConvention()` if we are going to use the default naming convention.

Reply:
Thanks for the suggestion. Revert the `MoreNamingConventions` and `MoreNamingConventionsTest` changes.